package.json

```diff
@@ -45,5 +51,12 @@
     "gettext.js": "^2.0.3",
     "jquery": "^3.5.1",
     "resumablejs": "^1.1.0"
-  }
+  },
+  "browserslist": [
+    "chrome 61",
+    "firefox 60",
+    "edge 16",
+    "opera 48",
+    "safari 11"
```

Comment:
The browser versions that were documented in #1497:

> * Chrome 64
> * Firefox 78
> * Microsoft Edge 79
> * Opera 51
> * Safari 11.1

I need to do some more testing if we're going with the older versions; I only went back to the newer versions quoted in this comment.